app/PlatformVerifier/Env.php

```diff
@@ -22,25 +22,46 @@ class Env
                 "See https://laravel.com/docs/5.8/queues for more information on queue drivers.",
     ];
 
-    public static function verifyRequirements()
+    public static function verifyRequirements($console = true)
```

Comment:
this works as a console thing too. 
Not sure it was really needed but it's neat to have it, I guess